docs/source/faq.rst

```diff
@@ -31,10 +34,53 @@ configured some templates to help and guide you through the process and
 hopefully to make it easier for you.
 
 Before submitting your bug, you can `search existing issues`__ to see if there
-is one open already.
+is one open already. If no existing issue covers your bug, please :ref:`have log
+excerpts ready <faq-logging>` that demonstrate the problem.
 
-Do not hesitate to :ref:`contact us<faq-contact-us>` if you are unsure about
+Do not hesitate to :ref:`contact us <faq-contact-us>` if you are unsure about
 your bug report.
 
 .. __: https://github.com/sopel-irc/sopel/issues/new/choose
 .. __: https://github.com/sopel-irc/sopel/issues
+
+
+.. _faq-logging:
+
+How to obtain logs?
+===================
+
+When requesting help, you'll probably be asked to provide logs illustrating the
+problem. If you :ref:`report a bug <faq-bug-report>`, the form includes a field
+for "Relevant logs" that you should fill in if possible.
+
+By default you will find Sopel's logs in the ``~/.sopel/logs`` folder. If Sopel's
+config file specifies the :attr:`~.config.core_section.CoreSection.homedir`
+and/or :attr:`~.config.core_section.CoreSection.logdir` setting(s), you will find
+the log files in ``/path/to/homedir/logdir``.
```

Comment:
I wonder if you could add a reference to the "install Sopel as a service", because in that install doc we suggest a specific place for Sopel's logs, that are more appropriate for a systemd service.

And by a reference I mean a link and a couple of words, not necessarily a full on `.. note`.